pkgs/development/libraries/nanoflann/default.nix

```diff
@@ -0,0 +1,22 @@
+{stdenv, fetchurl, cmake}:
+
+stdenv.mkDerivation {
+  name = "nanoflann-1.1.9";
+
+  src = fetchurl {
+    url = https://github.com/jlblancoc/nanoflann/archive/v1.1.9.tar.gz;
+    sha256 = "0y6nm5dypr4qhyq3mfn94s5l4v37vcl67ks52cxmkk8mfwmmi8d5";
+  };
+
+  buildInputs = [ cmake ];
+
+  doCheck = true;
+  checkTarget = "test";
+
+  meta = {
+    homepage = https://github.com/jlblancoc/nanoflann;
+    license = stdenv.lib.licenses.bsd3;
+    description = "Header only fork of flann, a fast approximate nearest neighbor implementation in high dimensional spaces";
```

Comment:
This is getting into bikeshedding, but to me, that the package is a fork of another does not belong in the short description. As a potential user, I mainly care what the library does for me.